js-packages/profiler-lib/src/cytograph.ts

```diff
@@ -369,6 +378,17 @@ export class CytographRendering {
                 'width': 'label',
             }
         },
+        {
```

Comment:
if you already have the diamond, why do you need the border too?
I think that in the future we'll use the border for something else, so perhaps it's better to save it